app/routes/settings+/profile.photo.tsx

```diff
@@ -211,14 +218,13 @@ export default function PhotoChooserModal() {
 					<ErrorList errors={form.errors} />
 				</Form>
 				<DialogClose asChild>
-					<Link
-						to=".."
-						preventScrollReset
+					<button
```

Comment:
I would definitely prefer to keep this a link because we are navigating. Is there a reason this was changed for a button?

Reply:
For the exit animation to display, the state of the dialogs `open` prop has to be changed to closed in close time-proximity to when the navigation is triggered, and this should happen the same way whether the cause of the exit is `onEscapeKeyDown`, `onPointerDownOutside` or clicking this `Icon` component. When I was writing this, it seemed easiest and most consistent to handle all of those events with the same `dismissModal` function that changed the state and then triggered navigation, and that navigation was redundant with the navigation triggered by the link, so I changed it to a button.

However, the following also seems to work:
```js
<DialogClose asChild>
	<Link
		to=".."
                onClick={() => setOpen(false)}
		preventScrollReset
		aria-label="Close"
		className="absolute right-20 top-20"
	>
		<Icon name="cross-1" />
	</Link>
</DialogClose>
```

Should I make this change?

Reply:
Yes. The benefit of this is that people can use it like a link (like right clicking and opening in a new tab etc). So I *always* put in the extra work to make all user-initiated navigations use a Link.

While you're at it, could you rebase on `main` and run `npm run format`?

Reply:
Sure thing